jobs/pulse3d/pulse3d-worker/src/lib/utils.py

```diff
@@ -22,23 +23,22 @@ def load_data_to_df(file_name, pr):
 
 
 def format_metadata(meta_sheet, pr, recording_length: int):
-    well_file = pr.wells[0]
-
+    first_available_well = next(iter(pr))
     return {
-        "plate_barcode": well_file.get(PLATE_BARCODE_UUID),
-        "recording_started_at": well_file[UTC_BEGINNING_RECORDING_UUID],
-        "file_format_version": well_file.version,
-        "instrument_serial_number": well_file.get(MANTARRAY_SERIAL_NUMBER_UUID),
+        "plate_barcode": first_available_well.get(PLATE_BARCODE_UUID, NOT_APPLICABLE_LABEL),
+        "recording_started_at": first_available_well[UTC_BEGINNING_RECORDING_UUID],
+        "file_format_version": first_available_well.version,
+        "instrument_serial_number": first_available_well.get(MANTARRAY_SERIAL_NUMBER_UUID),
         "length_microseconds": recording_length,
         "file_creation_timestamp": meta_sheet.iloc[11, 2],
         "mantarray_recording_session_id": uuid.uuid4(),
-        "uploading_computer_name": well_file.get(COMPUTER_NAME_HASH_UUID),
-        "acquisition_started_at": well_file.get(UTC_BEGINNING_DATA_ACQUISTION_UUID),
-        "session_log_id": well_file.get(
-            BACKEND_LOG_UUID, ""
-        ),  # this value is non-nullable in at least one table so need to return an empty string if not found
-        "software_version": well_file.get(SOFTWARE_RELEASE_VERSION_UUID),
-        "stim_barcode": well_file.get(STIM_BARCODE_UUID),
+        "uploading_computer_name": first_available_well.get(COMPUTER_NAME_HASH_UUID),
+        # fails if not a datetime
+        "acquisition_started_at": first_available_well.get(UTC_BEGINNING_DATA_ACQUISTION_UUID),
+        "session_log_id": first_available_well.get(BACKEND_LOG_UUID, ""),
+        # this value is non-nullable in at least one table so need to return an empty string if not found
```

Comment:
This comment should be attached to the line above. It's a little confusing to have it be underneath the line it's referring to